src/js/state/migrations/202101210823_renameToWorkspace.test.ts

```diff
@@ -0,0 +1,25 @@
+import getTestState, {getAllStates} from "../../test/helpers/getTestState"
+import migrate from "./202101210823_renameToWorkspace"
+
+test("migrating 202101210823_renameToWorkspace", () => {
```

Comment:
I think it would be good to add `expect.assertions(12)` (or whatever the number of expect calls we "expect"). Putting the expect calls in a loop is a little dangerous because if the array is empty, the expect calls won't get run, and the test will still pass.